modules/ROOT/pages/clauses/listing-functions.adoc

```diff
@@ -359,6 +363,7 @@ SHOW FUNCTIONS EXECUTABLE BY CURRENT USER YIELD *
 |===
 
 Notice that the two `roles` columns are empty due to missing the xref::administration/access-control/dbms-administration.adoc#access-control-dbms-administration-role-management[`SHOW ROLE`] privilege.
+Also note that the following columns are not present in the table: `signature`, `isBuiltIn`, `argumentDescription`, `returnDescription`, `aggregating`, and `isDeprecated`.
```

Comment:
This may be easier to read in a bulleted list: 

```suggestion
Also note that the following columns are not present in the table: 

* `signature`
* `isBuiltIn`
* `argumentDescription`
* `returnDescription`
* `aggregating`
* `isDeprecated`
```